src/transformers/testing_utils.py

```diff
@@ -227,6 +227,12 @@ def require_torch_non_multigpu(test_case):
         return test_case
 
 
+# this is a decorator identical to require_torch_non_multigpu, but is used as a quick band-aid to
+# allow all of examples to be run multi-gpu CI and it reminds us that tests decorated with this one
+# need to be ported and aren't so by design.
```

Comment:
what does ported mean in this comment?
Converted to support multi-gpu systems?

Reply:
> what does ported mean in this comment?

In my mind it means that a developer looked at the test and decided how it's supposed to operate and acted on it. It is either just fine as is - remove the fixme decorator, it's not fine - recode it to be fine and remove the fixme decorator.

Perhaps call it "_review_me" will be less ambiguous?

I originally started with the approach of explicitly listing tests that have been ported to multi-gpu.
@LysandreJik suggested we run all tests but those unported will get skipped to start with, hence this solution.